plonky/src/field_gl.rs

```diff
@@ -1,7 +1,7 @@
 #![allow(unused_imports)]
 use crate::ff::*;
 use core::ops::{Add, Div, Mul, Neg, Sub};
-#[derive(Eq)]
+#[derive(Eq, Serialize, Deserialize)]
```

Comment:
The serialize and Deserialize is useless here, cause it prints an element's Montegory format. Same to FrRepr